main.rb

```diff
@@ -1,4 +1,4 @@
-load "BST.rb"
+require_relative "BST.rb"
```

Comment:
In another file you have omitted `.rb`, keep it consistent according to the best practice.